src/Game.php

```diff
@@ -7,6 +7,10 @@
 use Monolog\Logger;
 
 use LotGD\Core\Models\Character;
+use LotGD\Core\Exceptions\ {
+    SceneNotFoundException,
+    ActionNotException
+};
```

Comment:
The PSR coding guideline allows this list notation of namespaces up to a depth of 2, meaning it could also be:

```php
 use LotGD\Core\ {
    Models\Character,
    Exceptions\SceneNotFoundException,
    Exceptions\ActionNotException
};
```
Do we want to adapt these guideline or restrict ourself to 1 level only?